exercises/sieve/instructions.md

```diff
@@ -1,28 +1,42 @@
 # Instructions
 
-Your task is to create a program that implements the Sieve of Eratosthenes algorithm to find prime numbers.
+Your task is to create a program that implements the Sieve of Eratosthenes algorithm to find all prime numbers less than or equal to a given number.
 
-A prime number is a number that is only divisible by 1 and itself.
+A prime number is a number larger than 1 that is only divisible by 1 and itself.
 For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
-
-The Sieve of Eratosthenes is an ancient algorithm that works by taking a list of numbers and crossing out all the numbers that aren't prime.
-
-A number that is **not** prime is called a "composite number".
+By contrast, 6 is _not_ a prime number as it not only divisible by 1 and itself, but also by 2 and 3.
 
 To use the Sieve of Eratosthenes, you first create a list of all the numbers between 2 and your given number.
 Then you repeat the following steps:
 
-1. Find the next unmarked number in your list. This is a prime number.
-2. Mark all the multiples of that prime number as composite (not prime).
+1. Find the next unmarked number in your list (skipping over marked numbers).
+   This must be a prime number.
```

Comment:
```suggestion
   This is a prime number.
```

As it is not immediately clear whether the «must be» is an inference or a requirement.